vcloud-blueprint-with-config.yaml

```diff
@@ -0,0 +1,277 @@
+###########################################################
```

Comment:
Blueprint file name should be *-blueprint.yaml (by convention).
In addition, our auto version system processes all blueprint files and updates imports etc to next Cloudify version when relevant.